docs: Mention about run-tests in contributing.md (#1255)
layer5io/meshery: CONTRIBUTING.md

Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`. One thing to note, you might NOT see the [Meshery UI](#contributing-ui) until the UI code is built as well.
After running Meshery server, you will need to select your **Cloud Provider** by navigating to `localhost:9081`. Only then you will be able to use the Meshery UI on port `3000`.

### Tests

> We now have ability to test some of the CI checks on your local machine using `make run-tests`.

Reviewer: ```suggestion
Users can now test their code on their local machine against the CI checks implemented using `make run-tests`.
```

Author: ok, making the changes now

> To test some of the CI checks on your local machine:

Reviewer: ```suggestion
To test code changes on your local machine, run the following command :
```
